uk.co.compucorp.civicrm.hrleaveandabsences/css/hrleaveandabsences.css

```diff
@@ -40,21 +40,27 @@
   font-weight: bold;
 }
 
+.work-pattern-calendar .work-pattern-week .week-days {
```

Comment:
I dont see any changes to the scss files. How come the css is changed?

Reply:
That file is not part of any sass pipeline, it's a plain old css file:

https://github.com/civicrm/civihr/blame/staging/uk.co.compucorp.civicrm.hrleaveandabsences/css/hrleaveandabsences.css

This file seems to contain changes for old admin pages before the sass pipeline was created. Since the work-pattern styles were already implemented on this file I made the changes here.